modules/users/manifests/add_relops_keys_to_relops_user.pp

```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
No need to add all user keys to the relops user.  It just needs the relops_common keys

Reply:
OK. Will change.